### PR TITLE
Feat/add uikit card detail

### PR DESCRIPTION
**Pull request by Thistle-Strum**
🔥 Summary
-Removes `DetailCard` and other components associated with `MobileBlocksCarousel` as they are not currently being used.

-Updates some styles associated with error messages and expanded `HashHeading`

### CODE REVIEW

**Review by valfaro123 — APPROVED** [review]

File: app/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ export const AccountDetailsCard: React.FC<AccountDetailsCardProps> = ({
 
   return (
     <>
-      <HeadContentContainer isTruncated={isTruncated}>
+      <HeadContentContainer isTruncated={isTruncated} isMobile={isMobile}>
```
Comment (DorianKwan) [inline app/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx]: Let's refrain from using this pattern.  We want to remove the `useAppWidth()` hook in the near future

File: app/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx [hunk 2]
```diff
@@ -171,9 +171,11 @@ const AccountDetailsWrapper = styled.div`
 
 const HeadContentContainer = styled(HeadContentWrapper)<{
   isTruncated?: boolean;
+  isMobile?: boolean;
 }>`
   margin: 0;
-  margin-bottom: ${({ isTruncated }) => (isTruncated ? '0.5rem' : '5rem')};
+  margin-bottom: ${({ isTruncated, isMobile }) =>
+    isTruncated || isMobile ? '0.5rem' : '5rem'};
```
Comment (DorianKwan) [inline app/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx]: Please refactor these styles to use media queries instead of `isMobile`

**Review by DorianKwan — CHANGES_REQUESTED** [review]

File: app/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx (hunk 1 above)
Comment (Thistle-Strum) [reply app/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx]: Removed!

File: app/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx (hunk 2 above)
Comment (Thistle-Strum) [reply app/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx]: Refactored!

**Review by DorianKwan — APPROVED** [review]

**Pull request merged**